common/constants.ts

```diff
@@ -68,5 +67,75 @@ function getAPIHost(): string {
 
 export const api = {
   host: getAPIHost(),
-  metricsHost: getMetricsHost()
+  metricsHost: getMetricsHost(),
+};
+
+export const staticEnvironmentPayload = {
```

Comment:
was thinking to put this in constants file since the uuids for shuttle data will be reused again for slack bot for equinox stats

Reply:
Thats fine, makes sense